doc/publishing.md

```diff
@@ -49,50 +51,135 @@ Each crate is "owned" by a list of maintainers, provided with the
+or metadata modifications to the corresponding crate.
+
+Other checks your submission will go through are:
+
+- It contains all required metadata.
+- It builds on all of our CI configurations.
+    - You can disable an unsupported target with the `available` property.
 
 ## Detailed steps
 
-These steps describe how to submit a new release. 
-
-1. Prepare a source archive (zip, tar) with the release sources and project
-   files.
-    1. This archive **must not** contain the `alire` directory generated by
-       `alr` in working directories. The `alire` directory is designed to be
-       put in your VCS ignore list.
-    1. The archive **must** contain a single directory (name not important)
-       containing, in turn, the sources. This is the kind of archives
-       automatically generated by GitHub, GitLab, Sourceforge... or through
-       `git archive`.
-1. Compute an integrity hash for the archive with `alr publish --hash <URL>`
-    1. The URL can be remote or local (`file://`), although using the final
-       online file is recommended.
-    1. The final online file should be immutable. Note that online
-       autogenerated archives can be found currently in the index, but should
-       be avoided whenever possible (because of lack of immutability
-       guarantees, see [this
-       conversation](https://github.community/t5/How-to-use-Git-and-GitHub/Ensuring-identical-tarballs-from-releases/m-p/34196/highlight/true#M8997)).
-       In the future, `alr publish` will offer support to generate and upload
-       such files more conveniently.
-1. Fill in the release manifest providing all the necessary information, 
-   dependencies (`depends-on`) and possibly excluded platforms
-   (`available`). See, for example, the [hello
-   world](https://github.com/alire-project/alire-index/blob/master/index/he/hello/hello-1.0.1.toml)
-   crate, or the [empty
-   skeleton](https://github.com/alire-project/alire-index/blob/master/templates/skeleton.toml)
-   template.
-1. Place the manifest file in the appropriate directory of your locally checked
-   out index repository, and create a new pull request. The detailed substeps
-   are the usual for submitting contributions in GitHub:
-    1. Fork the community index to your GitHub account.
-    1. Clone your fork locally and make the changes locally.
-    1. Commit and push the changes to your fork.
-    1. Create the pull request from your fork against the community repository
-       through the GitHub web interface (or the
-       [`hub`](https://github.com/github/hub) tool).
-        1. The base branch you select for the pull request will determine where
-           your changes will become available; see the section on [index
-           branches](#index-branches) for details.
+Depending on how you develop your project, you can use one of the following
+methods to prepare your release submission:
+
+### Starting from a git repository that contains an Alire workspace
+
```

Comment:
For both git options there should be an explanation about the list of allowed host and the fact the the commit must be on the host repo for the process to succeed.

Reply:
Do you mean, give a rationale for the list? Or just mention it too in the second git case?

Reply:
Provide both the list and a rationale.